### DISCUSSION TITLE
Keystone Blog - Any news?

### DISCUSSION THREAD

**Discussion by MinskLeo** [keystonejs/keystone#8716] [Questions]
Hi guys! Hopefully devs will take a look at this. The question is - is Keystone development alive? I see new releases in Github, that's great. Sometimes I see new stuff in docs marked as "New". But... there is no more blog posts here about new (or coming soon) features in Keystone Blog. The last post was created 14 of December, 2022. Almost a year without news on the website! Maybe I'm wrong and missing some other place, which is now primary used for news about Keystone. If so - please let me know where it is now. Otherwise - will new posts ever be posted on Keystone website? I really like Keystone, I like the progress which was made, but without making regular "reports" about what has been done, we can lost people from Keystone community. They will see that there is nothing happens on the blog from 2022 and will immediately leave. They will think that such a great tool is dead now...

https://keystonejs.com/blog
#KeystoneBlogBeAlive

**Comment by josemf** [comment]
I agree with @MinskLeo.

It is a lot of work to develop a framework and Tinkmill have been doing a great job with Keystone.

I think it is about building the community but also to keep the community using the framework. I'm not worried so much about KeystoneJS being super feature rich but people noticing work being done in terms of bug fixing, support and a clear roadmap, so people get their expectations right about using KeystoneJS.

**Comment by Thinkscape** [comment]
Roadmap has been quite stagnant as well... 
<img width="1113" alt="image" src="https://github.com/keystonejs/keystone/assets/270528/bafe5926-29e5-4061-b06c-15d0effbfe7f">

**Comment by MinskLeo** [comment]
Alright. Any news about news? 🙂 Sounds weird, I know, but everyone (including part of dev team) just leaves reaction emoji without any clarifying. Any responses will be highly appreciated!

**Comment by ChuckJonas** [comment]
It might be extreme to call this project "dead", but I think it's safe to say it's officially on "life support".

If I was choosing a platform to start a new product with in 2024, I don't think it would be wise to pick keystonejs...

**Reply by dcousens** [reply]
@ChuckJonas I understand your perspective, and development has slowed, don't get me wrong.
I am trying to maintain transparency into any ongoing work through descriptive pull requests, and I am always open to community feedback.

The website is stagnant, and needs an update to communicate to the wider community what a project roadmap could look like for 2024 and 2025.

**Reply by arihantverma** [reply]
Hey @dcousens thanks for the update and comment. I just wanted to ask… _is_ there any feature development happening? _Could_ look like, or _would_ look like?
